plugins/usbdmx/AsyncPluginImpl.h

```diff
@@ -60,10 +61,12 @@ class AsyncPluginImpl: public PluginImplInterface, public WidgetObserver {
    * @param plugin The parent Plugin object which is used when creating
    * devices.
    * @param debug_level the debug level to use for libusb.
+   * @param preferences The preferences container used by the plugin
    */
   AsyncPluginImpl(PluginAdaptor *plugin_adaptor,
                   Plugin *plugin,
-                  unsigned int debug_level);
+                  unsigned int debug_level,
+                  Preferences* preferences);
```

Comment:
Please move the * to the preferences so it's consistent with the rest of the file.